### DISCUSSION TITLE
lastUpdated?

### DISCUSSION THREAD

**Discussion by johnkomarnicki** [nuxt/content#1243] [Ideas]
l briefly recall Content V1 had some sort of timestamp on the markdown file for last time it was updated. Is there anyway currently to obtain this off the markdown file?

**Comment by Benbinbin** [comment]
I also want to figure out some methods to get this information from Git, and I think this feature provided from the core of content module will be more convenient.

**Comment by Benbinbin** [comment]
I find a relatvie issue for the content v1 version about timestamp for the content v1 version https://github.com/nuxt/content/issues/913 

It's impossible to get the correctly timestamp from Git when deploying the project?
